### ISSUE TITLE
[Menu design][Accessibility] Submenu descriptions are not read by VoiceOver

### ISSUE THREAD

**Issue by natastro** [mozilla-mobile/firefox-ios#23302]
## Prerequisites

- Turn on VoiceOver
- Turn on treatment-a of the iOS Menu Redesign
- Found on Firefox Beta iOS app 133.0
- Open any webpage

## Steps to reproduce

1. Open Firefox Beta iOS app
2. Open the menu from the hamburger menu icon button at the bottom of the screen
3. As the action sheet appears on screen, navigate to the Tools and Save submenus. Observe VoiceOver output

### Expected behavior / Actual behavior

According to the [Figma file](https://www.figma.com/design/RFz9fYtotQCQuinwcZujZt/%5B%5D-Menu-Redesign?node-id=19728-177609&t=rTRqRueB6nMFuifN-4), both submenus have a description that should be read aloud by VoiceOver

### Device & build information
* Device: iPhone 15, iOS 18.1.1
* Build version: Firefox Beta 133.0
* First seen version: First time I've seen this issue

### Notes
Attachments: 

https://github.com/user-attachments/assets/10e4e900-c088-4618-b892-fc5abc10737f

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-10648)

**Comment by data-sync-user** [comment]
➤ Matt P commented:

Natalie Stroud Winnie Teichmann This is what I currently see on nightly - does the caption look correct?

!Screenshot 2025-04-17 at 4.58.17 PM.jpeg|width=1206,height=2622,alt="Screenshot 2025-04-17 at 4.58.17 PM.jpeg"!
